src/utils.js

```diff
@@ -0,0 +1,71 @@
+function getUrlParms(url){
+  if (!url){
+    url = window.location.hash;
+  }
+
+  if (!url){
+    return {}
+  }
```

Comment:
what's going on here?

Reply:
in that function the `url` parameter can be optional, so you extract it from the current location, but you don't take the url, you just take the hash, which is the data after the `#` character, so probably the variables name are no the quite right, and if there is no data for `#` then there is no query string, the thing is that pkce, identity providers always redirect with that `#` character, so is a way of getting those tokens, @jhslashdb what do you think we can do to improve this in terms of semantics!?